daisy_workflows/image_import/import_image.sh

```diff
@@ -48,12 +51,37 @@ function resizeDisk() {
   fi
 }
 
-# Mount GCS bucket containing the disk image.
-mkdir -p /gcs/${SOURCEBUCKET}
-gcsfuse --implicit-dirs ${SOURCEBUCKET} /gcs/${SOURCEBUCKET}
+function copyImageToStagingDisk() {
+  local souceSizeGigabytes="$((1 + SOURCESIZE_BYTES / 1024 / 1024 / 1024))"
+  # We allocate double capacity here since the source file
+  # could be an OVA, which would require making an additional
+  # copy of its enclosed VMDK.
+  local stagingDiskSizeGigabytes="$((souceSizeGigabytes * 2))"
```

Comment:
I would guess that percentage of customers importing OVA as an image is very low. Additionally, we have instance import that is specifically designed for importing OVA/OVF files. 

We can still keep support for OVA in image import not to break backward compatibility. However, doubling scratch disk size for all regular/non-OVA imports (which are the bulk of imports) is a waste of resources. Can you resize the disk to double the capacity only if OVA is actually being imported?

Reply:
makes sense -- I'll add a check for the filetype and only double it if it's ova.